### ISSUE TITLE
Suppress Jruby message on OOM

### ISSUE THREAD

**Issue by ppf2** [elasticsearch/logstash#1496]
When LS runs out of memory, it writes out the following message:

```
"Error: Your application used more memory than the safety cap of 500M.            
Specify -J-Xmx####m to increase it (#### = cap size in MB).                      
Specify -w for full OutOfMemoryError stack trace""
```
This can be confusing to the end user given that -w is the flag for specifying worker threads in LS.

**Comment by untergeek** [comment]
It's dangerous to suppress this message, as it's the only indicator you may receive that your Logstash instance needs more memory.

While I agree that the `-w` may be confusing, we should instead document this behavior and instruct the end user what to do in the event this message is encountered.

**Comment by untergeek** [comment]
Closed in favor of #2415
